### DISCUSSION TITLE
can we  use rigtools to excute chrome API to enable / disable flags?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1331] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
??>

**Comment by akabutnicer** [comment]
yeah but you literally won't get anything from it

**Reply by ghost** [reply]
wdym , you can get inconigto  from the captive popup window flag, you can get  allow extension all chrome urls.

**Reply by ghost** [reply]
Nuh uh

**Reply by stuartcat** [reply]
ungate keep this I wanna see if I can use tr3nch 🙏

**Reply by jazzymakesstuff** [reply]
ryler can you give me a tut on this cuz I got chrome flags

**Reply by akabutnicer** [reply]
> wdym , you can get inconigto from the captive popup window flag, you can get allow extension all chrome urls.

to put this blatantly, you need extensions on all chrome URLs enabled for this idea to work currently...which is why I said what I said. furthermore, you can't remotely enable a flag from, say, the rigtools page; you would need to actually visit chrome://flags which completely defeats the purpose of an exploit.

**Comment by schoolexploitkid** [comment]
incognito isnt even worth it just use bookmarklet executor and history flood :P

**Reply by akabutnicer** [reply]
retarded statement but ok
